src/main/kotlin/me/zeroeightsix/kami/target/CategorisedTargets.kt

```diff
@@ -135,15 +139,34 @@ enum class ItemCategory(override val belongsFunc: (Item) -> Boolean) : Categoris
     // sorted by priority
     NONE({ false }),
     GRIEFING_TOOLS({ it in griefingTools }),
+
     AXES({ it is AxeItem }),
     SHOVELS({ it is ShovelItem }),
     PICKAXES({ it is PickaxeItem }),
     SWORDS({ it is SwordItem }),
     HOES({ it is HoeItem }),
     TOOLS({ it is ToolItem || it in additionalTools }),
+
     WEARABLE({ it is Wearable || (it as? BlockItem)?.block is Wearable }),
+
     FOOD({ it.isFood || it == Items.CAKE }),
+    ROTTEN_FOOD({ food ->
+        food.foodComponent?.statusEffects?.stream()
+            ?.anyMatch { effect -> !effect.first.effectType.isBeneficial }
+            ?: false
+    }),
+    NON_ROTTEN_FOOD({ food ->
+        food.foodComponent?.statusEffects?.stream()
+            ?.allMatch { effect -> effect.first.effectType.isBeneficial }
+            ?: false
+    }),
+
     REDSTONE({ it.group == ItemGroup.REDSTONE || it in additionalRedstone }),
+    RESOURCES({ it in resources }),
+    MOB_DROPS({ it in mobDrops }),
+    ORES({ (it as? BlockItem)?.block is OreBlock || it == Items.ANCIENT_DEBRIS }),
+    WOOD({ (it as? BlockItem)?.block?.defaultState?.material in woodMaterials || it in additionalWood }),
```

Comment:
What about things like wooden doors, trapdoors, maybe boats, maybe bookshelves, maybe crafting tables, fences, slabs, ...?

Reply:
all except the boat should be covered by the material check

Reply:
Good point. If the rest is covered by the material check, I'm happy.